implementation/jobberknoll/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM denoland/deno:2.1.4 AS builder
+
+WORKDIR /app
+
+COPY deno.json deno.lock api/deno.json app/deno.json core/deno.json infra/deno.json ./
+RUN deno install
+COPY . .
+RUN deno task compile
+
+
+FROM ubuntu:24.10 AS runner
+
+RUN apt-get -y update
+RUN apt-get -y install curl
+
+EXPOSE 8000
+WORKDIR /app
+USER ubuntu
+
+COPY --from=builder ./app/jobberknoll ./jobberknoll
+CMD ["./jobberknoll"]
```

At this point, after each code change, you need to redo the update and install curl, maybe these are not very expensive operations, but you can cache it.

```suggestion
FROM ubuntu:24.10 AS runner-base

RUN apt-get -y update
RUN apt-get -y install curl

WORKDIR /app
USER ubuntu

FROM denoland/deno:2.1.4 AS builder

WORKDIR /app

COPY deno.json deno.lock api/deno.json app/deno.json core/deno.json infra/deno.json ./
RUN deno install
COPY . .
RUN deno task compile

FROM runner-base AS runner

EXPOSE 8000

COPY --from=builder ./app/jobberknoll ./jobberknoll
CMD ["./jobberknoll"]
```

However, I have not tested it.

---

It is possible that `WORKDIR` and `USER` has to be in final `runner` stage

---

Fixed.